app/api/game.yaml

```diff
@@ -0,0 +1,163 @@
+swagger: '2.0'
+info:
+  version: 0.1.0
+  title: 'Torwolf Games API'
+paths:
+  /games:
+    get:
+      description: Finds games
+      parameters:
+        - name: offset
+          in: query
+          description: The offset at which the result set should begin.
+          required: false
+          type: integer
+          default: 0
+        - name: limit
+          in: query
+          description: Result will contain at most limit items
+          required: false
+          type: integer
+          default: 20
+        - name: phase
+          in: query
+          description: Games in the result set should be in the provided list of comma separated phases.
+          required: false
+          type: string
+          enum: ["FORMING", "STARTED", "COMPLETED"]
+        - name: name
+          in: query
+          description: Games should match the provided text search string
+          required: false
+          type: string
+      responses:
+        200:
+          description: Successful response
+          schema:
+            $ref: '#/definitions/ResultSet'
+        # TODO: error middleware
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    post:
+      description: Creates a game
+      parameters:
+        - name: game
+          in: body
+          description: Game to create
+          required: true
+          schema:
+            $ref: '#/definitions/Game'
+      responses:
+        200:
+          description: Successful response
+          schema:
+            $ref: '#/definitions/Game'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  /games/{id}:
+    get:
+      description: Gets a game
+      parameters:
+        - name: id
+          in: path
+          description: Id of the game to get
+          required: true
+          type: number
+          format: i32
+      responses:
+        200:
+          description: Successful response
+          schema:
+            $ref: '#/definitions/Game'
+        # TODO: error middleware
+        404:
+          description: Game was not found
+          schema:
+            $ref: '#/definitions/Error'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+
+definitions:
+  Game:
```

Comment:
Yeah, I do like that idea, but let's push it off to Asana for now and address it after 0.1 is released.